README.md

```diff
@@ -3,3 +3,44 @@
 String processing library for Urbit.
 
 Work in progress, feedback and pull requests welcome!
+
+### Installing / Using
+
+1. Copy lib/string.hoon into {pier}/{desk}/lib
+1. Copy app/string-tester.hoon into {pier}/{desk}/app
+1. From the dojo:
+  * |start %string-tester
+  * :string-tester %test
+
+You should see:
+```
+>=
+%sum
+%tail
+%get
+%starts-with
+%ends-with
+%contains
+%contains-any
+%contains-all
+%find-last
+%find-nth
+%find-first-any
+%find-last-any
+%trim-left
+%trim-right
+%trim
+%delete
+%delete-first
+%delete-last
+%delete-nth
+%delete-all
+%replace
+%replace-first
+%replace-last
+%replace-nth
+%replace-all
+%split
+%glue
```

Comment:
You're fine summarizing these into `%a-bunch-of-test-names` or something, for brevity.

Reply:
Yeah, I actually thought that, but the README isn't that big and this shows them exactly what they'll see so they'll know they did it right. Reasonable?

Reply:
Eh, you'd also need to keep this updated alongside the code, which is easy to miss.